code/backend/batch/utilities/tools/QuestionAnswerTool.py

```diff
@@ -83,7 +83,7 @@ def generate_on_your_data_llm_chain(
             if all((few_shot_example.values())):
                 examples.append(few_shot_example)
             else:
-                warnings.warn(
+                logger.warning(
```

Comment:
How come?

Reply:
When running the tests I get this outputted:
```
================================================================================== warnings summary ==================================================================================
code/tests/utilities/test_QuestionAnswerTool.py::test_correct_prompt_without_few_shot_example
  /workspaces/chat-with-your-data-solution-accelerator/code/backend/batch/utilities/tools/QuestionAnswerTool.py:86: UserWarning: Not all example fields are set in the config. Skipping few-shot example.
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=================================================================== 172 passed, 47 deselected, 1 warning in 6.82s ====================================================================
```

I thought just logging it at warning level would suffice. Would be happy to put it back though!